tests/e2e/util/image/image.go

```diff
@@ -33,14 +33,13 @@ var (
 const (
 	// TiDB Version
 	TiDBV3             = "v3.0.20"
-	TiDBLatestPrev     = "v5.0.3"
-	TiDBLatest         = "v5.1.0"
+	TiDBLatestPrev     = "v5.1.1"
```

Comment:
5.1.1 needs to be covered in L30.

Reply:
1063aa8067e6613585d68f96fdd73c90a9df15c2